fvm/src/call_manager/default.rs

```diff
@@ -415,6 +415,34 @@ where
         Ok(id)
     }
 
+    fn transform_into_eoa<K>(&mut self, addr: ActorID, actor: ActorState) -> Result<()>
```

Comment:
1. Call this `maybe_deploy_account`? In the future, it'll deploy other types.
2. Check if address is f410.

Reply:
(should also be logic in the executor and/or machine (likely the gas charging in the executor, and the functionality in the machine?)